docs/kubernetes-shortguide-definitions/container-definition-shortguide/index.md

```diff
@@ -16,4 +15,4 @@ show_on_rss_feed: false
 
 ### Container
 
-Similar to virtual machines, containers are light-weight isolated runtimes that share resources of the operating system without having to run a full operating system themselves. They consume few resources but contain a complete set of information needed to execute their contained application images such as files, environment variables, and libraries.
+Similar to virtual machines, containers are isolated runtimes that you can run your applications and services inside. Containers consume fewer resources than virtual machines, as they do not attempt to emulate a full operating system running on dedicated hardware. Instead, containers only bundle the files, environment variables, and libraries needed by the applications they run, and they share the other resources of the operating system that hosts them.
```

Comment:
VMs aren't particularly light-weight, so I reworded this to make that distinction clearer